core/model/src/driver.rs

```diff
@@ -352,6 +352,16 @@ impl Enter {
             token,
         }
     }
+
+    pub fn amount(&self) -> BigDecimal {
```

Comment:
Well I just followed the "convention" used all over the driver file :)
On the other hand - when the structs cover some kind of API contract it's safer to hide details and expose then via functions.

I'm leaving it as is.

Reply:
i fully agree with consistency :)

maybe adam can convince you ;) https://github.com/golemfactory/yagna/pull/1190#discussion_r604152823

Otherwise i'm fine with leaving it as is

Reply:
For consistency we should remove all these 'access' methods. It's un-rusty.

Reply:
Ok, I'm fine with removing the ones I added. I can issue another task for clearing the file from access functions entirely if you wish.

Reply:
It's fine.